freetextresponse/mixins.py

```diff
@@ -0,0 +1,32 @@
+"""
+Mixins for the Free Text Response XBlock
+"""
+import datetime
+import pytz
+from django.conf import settings
+
+
+class EnforceDueDates(object):
+    """
+    xBlock Mixin to allow xblocks to check the due date
+    (taking the graceperiod into account) of the
+    subsection in which they are placed
+    """
+    # get instance timezone
+    self.site_tz = pytz.timezone(getattr(settings, 'TIME_ZONE', pytz.utc.zone))
+
+    # Default values pulled from platform to None for tests
+    due = None
+    graceperiod = None
```

Comment:
This pulls value from Grading page in studio (matches CAPA). Needs to be set here to `None` for tests.